### ISSUE TITLE
consider increase "engine.node" in package.json because of using naitve BigInt

### ISSUE THREAD

**Issue by rocka** [acrisci/node-dbus-next#7]
Currently, `engine.node` in packege.json file is

https://github.com/acrisci/node-dbus-next/blob/96ef3f49cd8f03241eda44aba4129555e3ed422d/package.json#L59-L61

and we are using BigInt iteral here

https://github.com/acrisci/node-dbus-next/blob/96ef3f49cd8f03241eda44aba4129555e3ed422d/lib/constants.js#L1-L5

But according to [node.green](https://node.green/#ESNEXT-candidate--stage-3--BigInt) , BigInt isn't supported until Node 10.8.0. I think we definitely should change this.

**Comment by acrisci** [comment]
Yeah definitely should be changed.

I will take out the `0n` notation and put in a polyfill to lower the node version to less than this.

**Comment by acrisci** [comment]
I successfully ran my test suite on `8.2.1`, so I set the engine to later than that version.

It will still use BigInt by default, but I put in a compatibility mode configuration option. This is required for versions between `8.2.1` and `10.8.0`.

```javascript
const dbus = require('dbus-next');
dbus.setBigIntCompat(true);
```

This will use [JSBI](https://github.com/GoogleChromeLabs/jsbi) for long types.

I added [this test](https://github.com/acrisci/node-dbus-next/blob/master/test/integration/long-compat.test.js) to test compatibility mode.
